src/plugins/intel_cpu/thirdparty/ACLConfig.cmake

```
@@ -351,38 +353,53 @@ elseif(NOT TARGET arm_compute::arm_compute)
     else()
         set(arm_compute build/${OV_CPU_ARM_TARGET_ARCH}/libarm_compute-static.a)
     endif()
-    set(arm_compute_full_path "${ARM_COMPUTE_SOURCE_DIR}/${arm_compute}")
 
     list(APPEND ARM_COMPUTE_OPTIONS fixed_format_kernels=True)
 
-    add_custom_command(
-        OUTPUT
-            ${arm_compute_full_path}
-        COMMAND ${CMAKE_COMMAND} -E env ${cmake_build_env}
-            ${SCONS} ${ARM_COMPUTE_OPTIONS}
-                ${arm_compute}
-        WORKING_DIRECTORY ${ARM_COMPUTE_SOURCE_DIR}
-        COMMENT "Build Arm Compute Library"
-        DEPENDS ${SOURCES}
-                ${CMAKE_CURRENT_LIST_FILE}
-                ${ARM_COMPUTE_SOURCE_DIR}/SConscript
-                ${ARM_COMPUTE_SOURCE_DIR}/SConstruct)
-
-    # Compute Library uses cppthreads=1
-    # if one day will rely on TBB only, we can omit this dependency
+    # Find Threads - required by ACL
     find_package(Threads REQUIRED)
 
-    # Import targets
+    # Convert list to space-separated string for ExternalProject
+    string(REPLACE ";" " " ARM_COMPUTE_OPTIONS_STR "${ARM_COMPUTE_OPTIONS}")
+
+    # Use ExternalProject to build ACL
+    ExternalProject_Add(arm_compute_build
+        PREFIX ${ARM_COMPUTE_BUILD_DIR}
+        SOURCE_DIR ${ARM_COMPUTE_SOURCE_DIR}
+        CONFIGURE_COMMAND ""
+        BUILD_COMMAND ${CMAKE_COMMAND} -E env ${cmake_build_env}
+            ${SCONS} ${ARM_COMPUTE_OPTIONS}
+                ${arm_compute}
+        BUILD_IN_SOURCE 1
+        BUILD_BYPRODUCTS ${ARM_COMPUTE_BUILD_DIR}/${arm_compute}
+        INSTALL_COMMAND ${CMAKE_COMMAND} -E make_directory ${ARM_COMPUTE_BUILD_DIR}/build/${OV_CPU_ARM_TARGET_ARCH}
+            COMMAND ${CMAKE_COMMAND} -E make_directory ${ARM_COMPUTE_BUILD_DIR}/build/${OV_CPU_ARM_TARGET_ARCH}/src/core
+            COMMAND ${CMAKE_COMMAND} -E copy
+                ${ARM_COMPUTE_SOURCE_DIR}/build/${OV_CPU_ARM_TARGET_ARCH}/libarm_compute-static.a
+                ${ARM_COMPUTE_BUILD_DIR}/build/${OV_CPU_ARM_TARGET_ARCH}/libarm_compute-static.a
+            COMMAND ${CMAKE_COMMAND} -E copy
+                ${ARM_COMPUTE_SOURCE_DIR}/build/${OV_CPU_ARM_TARGET_ARCH}/src/core/arm_compute_version.embed
+                ${ARM_COMPUTE_BUILD_DIR}/build/${OV_CPU_ARM_TARGET_ARCH}/src/core/arm_compute_version.embed
+            COMMAND ${CMAKE_COMMAND} -E remove_directory ${ARM_COMPUTE_SOURCE_DIR}/build
+        LOG_BUILD ON
+    )
 
-    add_custom_target(arm_compute_static_libs DEPENDS ${arm_compute_full_path})
+    # Get the full path to the built library after it's copied
+    set(arm_compute_full_path "${ARM_COMPUTE_BUILD_DIR}/${arm_compute}")
+    
+    message(STATUS "ACL build configuration:")
+    message(STATUS "  ARM_COMPUTE_BUILD_DIR: ${ARM_COMPUTE_BUILD_DIR}")
+    message(STATUS "  arm_compute: ${arm_compute}")
+    message(STATUS "  arm_compute_full_path: ${arm_compute_full_path}")
```

Consider hiding it, otherwise it is gonna be quite verbose
```suggestion
    message(DEBUG "ACL build configuration:")
    message(DEBUG "  ARM_COMPUTE_BUILD_DIR: ${ARM_COMPUTE_BUILD_DIR}")
    message(DEBUG "  arm_compute: ${arm_compute}")
    message(DEBUG "  arm_compute_full_path: ${arm_compute_full_path}")
```